packages/ai/src/evals/eval.ts

```diff
@@ -105,19 +110,36 @@ async function registerEval<
   // TODO: EXPERIMENTS - we were creating `evalScope` here before
 
   // check if user passed a specific baseline id to the CLI
-  const baselineId = inject('baseline');
+  const providedBaselineId = inject('baseline');
 
-  const result = await describe(
+  const result = describe(
     `evaluate: ${evalName}`,
     async () => {
       const dataset = await datasetPromise;
-      // if user passed a baseline id, if not, find the latest evaluation and use it as a baseline
-      const baseline = baselineId
-        ? await findEvaluationCases(baselineId)
-        : await findBaseline(evalName);
+      const { dataset: datasetName, region, apiUrl, token } = getEvaluationApiConfig();
       // create a version code
       const evalVersion = nanoid();
       let evalId = ''; // get traceId
+      let anyCaseFailed = false;
+      const suiteStart = performance.now();
+      let successCases = 0;
```

Comment:
in `afterAll()` we have access to the suite along with its children, I would say its safer to loop over the suite tasks and check state of each instead of counting them this way. 

Another q: are these numbers going to be used in the UI?